projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/quote/quote.e2e-flaky.cy.ts

```diff
@@ -23,6 +24,17 @@ const buyer = {
 };
 
 context('Quote', () => {
+  // before all tests - ensure that cart is empty
+  before(() => {
```

Comment:
why only in a before and not a beforeEach as done in the quote-configurator.e2e-flaky? 
so the lines 
`  quote.addProductToCart(TEST_PRODUCT_HAMMER_DRILLING_ID, '1');
    cart.clearActiveCart();
    cart.validateEmptyCart();`

could be done right before `registerReadQuoteRoute()`. 
The whole before() block could be removed then.